docker-compose.override.yml

```diff
@@ -15,6 +15,6 @@ services:
     ports:
       - "80"
       - "443"
-    volumes:
-      - ${APPDATA}/Microsoft/UserSecrets:/root/.microsoft/usersecrets:ro
-      - ${APPDATA}/ASP.NET/Https:/root/.aspnet/https:ro
\ No newline at end of file
+    extends:
+      file: windows.yml
+      service: certvolumes
```

Comment:
These volumes are nested under a service. Can they be moved to just a volume object?

Reply:
The extends keyword seems to be needing a service to reference.